granulate_utils/metadata/cloud.py

```diff
@@ -44,6 +44,7 @@ class GcpInstanceMetadata(InstanceMetadataBase):
     instance_id: str
     image_id: str
     name: str
+    region: str = None
```

Comment:
Why `= None`?

Reply:
Actually, I can remove it. It's a leftover.